test/sharness/t0087-repo-robust-gc.sh

```diff
@@ -0,0 +1,143 @@
+#!/bin/sh
+#
+# Copyright (c) 2016 Jeromy Johnson
+# MIT Licensed; see the LICENSE file in this repository.
+#
+
+test_description="Test ipfs pinning operations"
+
+. lib/test-lib.sh
+set -e
```

Comment:
I can remove it if you want, but note it does serve a very useful purpose.  Please see #3721.  I already know about `-i` and that is _not_ why i am using it.